jest.config.js

```diff
@@ -1,7 +1,8 @@
 /** @type {import('ts-jest').JestConfigWithTsJest} */
-
 module.exports = {
-  coveragePathIgnorePatterns: ['/dist', '/node_modules'],
+  coverageDirectory: 'coverage',
+  coveragePathIgnorePatterns: ['<rootDir>/dist', '<rootDir>/node_modules'],
+  coverageReporters: ['text-summary'],
```

Comment:
Is this line required for it to work again or could we bring back the table view?

Reply:
Changed this to `text` and it brought back the table view.

![Screen Shot 2023-02-01 at 11 19 38 AM](https://user-images.githubusercontent.com/33762/216142082-5ee1a1d7-69df-44fd-b995-12a45453b7b7.png)

You're just going to lose the `lcov` reporter that generates the fancy HTML page but I don't know if you ever used that locally or not. https://istanbul.js.org/docs/advanced/alternative-reporters/